render-release-notes.sh

```diff
@@ -0,0 +1,174 @@
+#!/bin/bash
+
+me=$(basename $0)
+
+function showUsage() {
```

Comment:
The convention in Git's source code is to skip the `function` keyword, to add a space before `()` and to call the function `usage`:

```sh
usage () {
...
}
```